spec/features/idv/analytics_spec.rb

```diff
@@ -434,6 +542,82 @@
     end
   end
 
+  context 'Happy hybrid path' do
+    before do
+      allow(Telephony).to receive(:send_doc_auth_link).and_wrap_original do |impl, config|
+        @sms_link = config[:link]
+        impl.call(**config)
+      end.at_least(1).times
+
+      perform_in_browser(:desktop) do
+        sign_in_and_2fa_user(user)
+        visit_idp_from_sp_with_ial2(:oidc)
+        complete_welcome_step
+        complete_agreement_step
+        click_send_link
+      end
+
+      perform_in_browser(:mobile) do
+        visit @sms_link
+        attach_and_submit_images
+        visit idv_hybrid_mobile_document_capture_url
+      end
+
+      perform_in_browser(:desktop) do
+        click_idv_continue
+        visit idv_ssn_url
+        complete_ssn_step
+        complete_verify_step
+        fill_out_phone_form_ok('202-555-1212')
+        verify_phone_otp
+        complete_enter_password_step(user)
+        acknowledge_and_confirm_personal_key
+      end
+    end
+
+    it 'records all of the events' do
```

Comment:
We can also reuse this for hybrid or non hybrid happy path as shared example, but not a blocker.

LGTM.